Userland/Libraries/LibJS/Runtime/PropertyName.h

```diff
@@ -176,12 +176,13 @@ class PropertyName {
         return StringOrSymbol(as_symbol());
     }
 
+    // Related to https://tc39.es/ecma262/#sec-ispropertykey
```

Comment:
Do we want to mention the spec like this?
I put it here since it could be confusing why number -> string but symbol can be a symbol

Reply:
I don't see how it's related - `is_valid()` is basically our `IsPropertyKey()` since we also accept numeric ones.